apps/cli/src/utils/streaming-display.ts

```
@@ -0,0 +1,92 @@
+          const usageText = `\n\n[Model: ${usage.model} | Tokens: ${usage.totalTokens} (${usage.promptTokens}+${usage.completionTokens})]`;
+          process.stdout.write(usageText);
+        }
+        break;
+
+      case 'done':
+        this.isComplete = true;
+        process.stdout.write('\n');
+        break;
+
+      case 'error':
+        const errorMsg = `${this.options.errorPrefix} ${event.data.message}\n`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)